partials/tabs/getinvolved.html

```diff
@@ -6,7 +6,7 @@
 
 
         <div class="col-md-8 col-md-offset-2">
-            <div class="title-get-involved" >get involved</div>
+            <div class="title-get-involved" >Get Involved</div>
```

Comment:
Tried the changes locally, Actually its a part of style philosophy followed throughout the website that headings are in lowercase (just like coala is always spelled with a lowercase c). So we can leave the headings at it is.